forge/test/operators/utils/compat.py

```diff
@@ -22,6 +22,85 @@ class TestDevice:
     pass
 
 
+class TestTensorsUtils:
+
+    dev_data_format_to_dtype = {
+        # forge.DataFormat.Bfp2: torch.float16,
+        forge.DataFormat.Bfp2: torch.bool,  # mapped
```

Comment:
Bfp data formats represent floating point numbers, not boolean values. Therefore I think we should have here some float...

Reply:
Any thoughts on this @vbrkicTT

Reply:
I think we clarified this today with @nvukobratTT.
You are right that it's wrong to map floating point to boolean. I followed mapping from pytorch_dtype_to_forge_dataformat which was probably completely misleading to do here. The types of input tensors should be as close as possible between forge and torch types.

Reply:
no booleans anymore, only floats and integer for now